### ISSUE TITLE
label_smoothing实现与原论文不一致

### ISSUE THREAD

**Issue by samuellees** [mindspore-ai/mindspore#73]
mindspore中将softmax+crossentropy+one_hot+label_smoothing合并实现了，对应SoftmaxCrossEntropyWithLogits算子。
其中关于label smoothing部分，on_value和off_value计算代码如下：
`# mindspore\mindspore\nn\loss\loss.py`
`self.one_hot = P.OneHot()`
`self.on_value = Tensor(1.0 - self.smooth_factor, mstype.float32)`
`self.off_value = Tensor(1.0 * self.smooth_factor / (self.num_classes - 1), mstype.float32)`

这里均匀分布取了K=1/(num_classes -1)，按照[inception v3论文](https://arxiv.org/pdf/1512.00567.pdf)的计算方式，应为
`self.on_value = Tensor(1.0 - smooth_factor + smooth_factor / num_classes, mstype.float32)`
`self.off_value = Tensor(1.0 * smooth_factor / num_classes, mstype.float32)`

另外，[TensorFlow中的计算方式](https://tensorflow.google.cn/versions/r1.15/api_docs/python/tf/losses/softmax_cross_entropy)与论文提出的计算方式一致。

以下是类似函数的路径：
mindspore\nn\loss\loss.py，SoftmaxCrossEntropyWithLogits
model_zoo\official\cv\mobilenetv2\train.py，CrossEntropyWithLabelSmooth
model_zoo\official\cv\mobilenetv2_quant\src\utils.py，CrossEntropyWithLabelSmooth
model_zoo\official\cv\mobilenetv3\train.py， CrossEntropyWithLabelSmooth

**Comment by buxue0727** [comment]
smooth factor实际上代表的是1 - on_value。当前参数上用smooth factor表示1-on_value表示确实造成了歧义，我们正在修改为标准的论文计算方式。非常感谢您的意见。

当前您可以按照以下理解来使用：
其实我们这个求和也是1，只是计算方式不一样，smooth factor和tf的smooth factor表达的意思不一样，我们的是on value。
郭琦(g00395938) 2020-08-17 17:00
比如smooth factor是0.1的时候，我们的onvalue就是0.9，tf的是0.9 + 0.9/1000
